internal/provider/kubernetes/config/default/auth_proxy_patch.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: envoy-gateway
-  namespace: system
```

Comment:
This change is unneeded due to https://github.com/envoyproxy/gateway/blob/main/internal/provider/kubernetes/config/default/kustomization.yaml#L1-L2